<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::StrokeEditorWidget</class>
 <widget class="QWidget" name="GfxPaint::StrokeEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="proportionalSpacingHeightSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.062500000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QSpinBox" name="dabCountSpinBox">
     <property name="specialValueText">
      <string>Use Spacing</string>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="dabCountLabel">
     <property name="text">
      <string>&amp;Dab Count</string>
     </property>
     <property name="buddy">
      <cstring>dabCountSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="proportionalSpacingWidthSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.062500000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="spaceLabel">
     <property name="text">
      <string>&amp;Space</string>
     </property>
     <property name="buddy">
      <cstring>spaceComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="absoluteSpacingWidthSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="spaceComboBox">
     <item>
      <property name="text">
       <string>Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Object Aspect-corrected</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>World</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>View</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="absoluteSpacingHeightSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>512.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="continuousCheckBox">
     <property name="text">
      <string>&amp;Continuous</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="proportionalSpacingLabel">
     <property name="text">
      <string>&amp;Proportional Spacing</string>
     </property>
     <property name="buddy">
      <cstring>proportionalSpacingWidthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="absoluteSpacingLabel">
     <property name="text">
      <string>&amp;Absolute Spacing</string>
     </property>
     <property name="buddy">
      <cstring>absoluteSpacingWidthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="metricLabel">
     <property name="text">
      <string>&amp;Metric</string>
     </property>
     <property name="buddy">
      <cstring>metricComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="metricComboBox">
     <item>
      <property name="text">
       <string>Euclidean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manhattan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chebyshev</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spaceComboBox</tabstop>
  <tabstop>metricComboBox</tabstop>
  <tabstop>continuousCheckBox</tabstop>
  <tabstop>absoluteSpacingWidthSpinBox</tabstop>
  <tabstop>absoluteSpacingHeightSpinBox</tabstop>
  <tabstop>proportionalSpacingWidthSpinBox</tabstop>
  <tabstop>proportionalSpacingHeightSpinBox</tabstop>
  <tabstop>dabCountSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
